features/support/env.rb

```diff
@@ -103,7 +103,7 @@
 end
 
 # Force deprecations to raise an exception.
-ActiveSupport::Deprecation.behavior = :raise
+ActiveSupport::Deprecation.behavior = ActiveAdmin::Dependency.rails?('< 5.1.x') ? :raise : :stderr # FIXME: need database_cleaner 1.6.0.
```

Comment:
@varyonic , basically  I'm ok with both options, but we should somehow not forget to return  ```ActiveSupport::Deprecation.behavior```